spec/api-app-spec.js

```diff
@@ -87,6 +87,30 @@ describe('app module', function() {
     });
   });
 
+  describe('app.addDistributedNotificationObserver', function() {
+    it('emits custom events for notifications', function(done) {
+      if (process.platform != 'darwin')
+        done();
+
+      this.timeout(5000);
+
+      app.addDistributedNotificationObserver(
+        'ExampleNotification',
+        'example-notification'
+      );
+
+      app.on('example-notification', function(){
+        done();
+      });
+
+      //Emit an event
+      ChildProcess.exec('echo \'import Foundation;' +
+        'NSDistributedNotificationCenter.defaultCenter().postNotificationName'+
+        '("ExampleNotification", object: nil);\' | swift -'
```

Comment:
I'm pretty sure we don't have access to the `swift` command on Travis-CI. I don't know if there would be another way of posting a notification.